code/game/gamemodes/antag_spawner.dm

```diff
@@ -205,7 +205,36 @@
 
 
 ///////////SLAUGHTER DEMON
+/obj/item/weapon/antag_spawner/slaughter_demon
+	name = "Bottle of Blood"
```

Comment:
names should not be capsed,because when examined it'll say "This is the Bottle of Blood."